core/src/main/java/com/linecorp/armeria/common/stream/StreamMessage.java

```diff
@@ -1073,4 +1074,33 @@ default InputStream toInputStream(Function<? super T, ? extends HttpData> httpDa
         requireNonNull(executor, "executor");
         return new StreamMessageInputStream<>(this, httpDataConverter, executor);
     }
+
+    /**
+     * Emits the last value depending on whether this {@link StreamMessage}
+     * completes successfully or exceptionally.
```

Comment:
I think it doesn't emit the value depending on whether it is completed successfully or exceptionally. It always emits a value. 😉 
How about removing `depending on`?

Reply:
✅ updated!